templates/field/field--node--oe-person-transparency-links--oe-person--full.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
```

Comment:
<img width="1091" alt="Screenshot 2021-03-09 at 12 32 40" src="https://user-images.githubusercontent.com/1183951/110449902-b6e75200-80d3-11eb-8201-20c5c6d5226d.png">

Transparency links field is a link field that can not be built as on mockup.